pom.xml

```
@@ -15,7 +15,11 @@
   <name>book-store</name>
   <description>book-store</description>
   <properties>
-    <java.version>21</java.version>
+    <java.version>17</java.version>
+    <maven.checkstyle.plugin.version>3.1.1</maven.checkstyle.plugin.version>
+    <maven.checkstyle.plugin.configLocation>
+      https://raw.githubusercontent.com/mate-academy/style-guides/master/java/checkstyle.xml
```

let's replace the web address with the file that we have locally in our project
```suggestion
      checkstyle.xml
```